src/multilayerqg.jl

```diff
@@ -561,11 +606,12 @@ is the number of layers in the fluid. (When ``n=1``, only the kinetic energy is
 
 The kinetic energy at the ``j``-th fluid layer is 
 ```math
-\\textrm{KE}_j = \\frac{H_j}{H} \\int \\frac1{2} |\\boldsymbol{\\nabla} \\psi_j|^2 \\frac{\\mathrm{d}^2 \\boldsymbol{x}}{L_x L_y} \\ , \\quad j = 1, \\dots, n \\ ,
+𝖪𝖤_j = \\frac{H_j}{H} \\int \\frac1{2} |{\\bf ∇} ψ_j|^2 \\frac{𝖽x 𝖽y}{L_x L_y} = \\frac1{2} \\frac{H_j}{H} \\sum_{𝐤} |𝐤|² |ψ̂_j|², \\ j = 1, ..., n \\ ,
```

Comment:
I like the simplifications you've achieved here. 

I have one very small comment. I noticed that there is some notation that is not simple roman letters. This makes sense for math notation and vectors, but is much more subtle when (unicode?) roman letters are used in place of normal roman letters. For example in this equation I believe ``KE`` and differential ``d`` are unicode, but this was only apparent to me through the font changes in the rendered equations (see below). I think that is okay for now, but it may be useful to make unicode usage more obvious (is that even possible?) if a wider group of people start modifying the docs.

<img width="544" alt="Screen Shot 2020-12-28 at 9 10 54 AM" src="https://user-images.githubusercontent.com/43418592/103231517-cd5f7900-48ec-11eb-8ced-177b0897b306.png">

Reply:
Good point. I mostly chose to use unicode because then the docstrings look nice in the REPL... Otherwise you see things like, e.g. \textrm{KE}. 

But you make a nice point. I'll raise an issue to have a small section "Contributors" or something like that and add this clarification...

@BrodiePearson did you have a look at the Stochastic Forcing section? I'm still not 100% satisfied, e.g. a definition of W_t should appear. But at some point one needs to stop since we are not writing a book here... :)

Reply:
(see #180)